ui/components/MesheryMeshInterface/PatternService/RJSFCustomComponents/ObjectFieldTemplate.js

```diff
@@ -136,7 +136,14 @@ const ObjectFieldTemplate = ({
           <Grid
             item={true}
             sm={12}
-            lg={12} //@harkiratsm, please recheck the logic once again. I'm deleting it now to fix problems with the RJSF
+            lg={
+              element.type === "object" ||
+              element.type === "array" ||
+              element.__additional_property ||
+              additional
```

Comment:
I see why you have used type "object" in `allOf`, `oneOf` etc. 
But schema modification comes at cost.(We saw it while we moved from draft04 to draft07)